### PR TITLE
Harmony: move to Openpype

### PR DESCRIPTION
**Pull request by kalisp**
## Brief description
Moved Harmony from avalon-core to Harmony. 

## Additional info
Cleaned up logic for sending logs to Tray.
Updated OpenHarmony to current version.

Requires: https://github.com/pypeclub/avalon-core/pull/419

Still in testing!!!

### CODE REVIEW

**Comment by mkolar** [comment]
Task linked: [OP-2414 Move harmony to openpype](https://app.clickup.com/t/6658547/OP-2414)

File: openpype/hooks/pre_non_python_host_launch.py [hunk 1]
```diff
@@ -51,3 +52,17 @@ def execute(self):
 
         if remainders:
             self.launch_context.launch_args.extend(remainders)
+
+        if platform.system().lower() == "windows":
+            # expected behavior - openpype_console opens window with logs
+            # openpype_gui has stdout/stderr available for capturing
+            if "openpype_gui" in os.environ.get("OPENPYPE_EXECUTABLE"):
```
Comment (iLLiCiTiT) [inline openpype/hooks/pre_non_python_host_launch.py:59]: Just make sure that it is looked for `openpype_gui` only in filename.
```suggestion
            executable_path = os.environ.get("OPENPYPE_EXECUTABLE")
            executable_filename = ""
            if executable_path:
                executable_filename = os.path.dirname(
                    os.environ.get("OPENPYPE_EXECUTABLE")
                )
            if "openpype_gui" in executable_filename:
```

File: openpype/tools/stdout_broker/app.py [hunk 2]
```diff
@@ -62,207 +65,152 @@ class ConsoleTrayApp:
...
             "host": self.host_id,
-            "action": host_console_listener.MsgAction.ADD,
-            "text": "\n".join(lines)
+            "action": MsgAction.INITIALIZED,
+            "text": "Integration with {}".format(
+                str.capitalize(self.host_name))
         }
-
         self._send(payload)
+        self.loop_timer.start()
```
Comment (iLLiCiTiT) [inline openpype/tools/stdout_broker/app.py:149]: Why don't you start look timer on init?

File: openpype/hooks/pre_non_python_host_launch.py [hunk 3]
```diff
@@ -51,3 +51,7 @@ def execute(self):
 
         if remainders:
             self.launch_context.launch_args.extend(remainders)
+
+        self.launch_context.kwargs = \
+            get_non_python_host_kwargs(self.launch_context.kwargs)
```
Comment (iLLiCiTiT) [inline openpype/hooks/pre_non_python_host_launch.py:56]: Logic from `get_non_python_host_kwargs` is used only here. I would not move it to lib. You are moving function for single use from almost empty file to overfilled file...

File: openpype/lib/execute.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import os
 import subprocess
 import distutils.spawn
+import platform
```
Comment (iLLiCiTiT) [inline openpype/lib/execute.py:4]:
```suggestion
```

File: openpype/lib/execute.py [hunk 5]
```diff
@@ -272,3 +273,4 @@ def get_linux_launcher_args(*args):
         launch_args.extend(args)
 
     return launch_args
+
```
Comment (iLLiCiTiT) [inline openpype/lib/execute.py:276]:
```suggestion
```

**Review by iLLiCiTiT — APPROVED** [review]
Code looks ok. Please cleanup hound suggestions in python files (he's right) and remove unused imports. @64qam can you test functionality please?

**Review by 64qam — APPROVED** [review]

**Pull request merged**